src/test/java/com/jcabi/http/wire/LastModifiedCachingWireTest.java

```diff
@@ -65,6 +68,11 @@
     private static final String BODY_UPDATED = "Test body updated";
 
     /**
+     * Test body updated 2.
+     * */
+    private static final String BODY_UPDATED_2 = "Test body updated 2";
```

Comment:
@mkordas No as Qulice insists on this, and I have merely copied the style already in the tests.